.travis.yml

```diff
@@ -38,4 +38,5 @@ script:
   - "export SPARK_HOME=./spark-2.1.0-bin-hadoop2.7/"
   - "export JAVA_HOME=/usr/lib/jvm/java-8-oracle"
   - cd geopyspark-backend && ./sbt "project geotrellis-backend" assembly
-  - "cd .. && pytest"
+  - cd .. && pytest -k "not methods"
+  - pytest geopyspark/tests/tile_layer_methods_test.py
```

Comment:
@echeipesh I had to make the `TileLayerMethodsTest` it's own separate test for `pytest` because, for some reason, it will fail if it's run after the `TileLayerMetadataTest`. I'm not sure why this is the case, but I've decided to come back to the issue at a later time.